transport/src/main/java/io/netty/channel/DefaultChannelConfig.java

```diff
@@ -302,6 +319,36 @@ public ChannelConfig setRecvByteBufAllocator(RecvByteBufAllocator allocator) {
         return this;
     }
 
+    public boolean isAutoFlush() {
+        return autoFlush == 1;
+    }
+
+    public ChannelConfig setAutoFlush(boolean autoFlush) {
```

Comment:
@NiteshKant btw you will also need to override this method on all sub-classes and return the correct config type